config/crdroid.mk

```diff
@@ -177,3 +177,6 @@ PRODUCT_PACKAGES += \
     QStileTeardrop \
     QStileTriangle \
     QStileWavey
+
+# Customizable volume panels
+-include packages/apps/Plugins/plugins.mk
```

Comment:
where is this?

Reply:
https://github.com/PotatoProject/packages_apps_Plugins